Modules/Invoice/Resources/views/modals/invoice-reminder.blade.php

```diff
@@ -43,7 +43,7 @@ class="form-control" required>
                                     <i class="fa fa-question-circle"></i>
                                 </span>
                             </label>
-                            <input type="text" name="bcc" id="bcc" class="form-control" value="">
+                            <input type="email" name="bcc" id="bcc" class="form-control" value="{{ config('invoice.mail.send-invoice.email') }}">
```

Comment:
Already via CC, the mail is sent to the same email we don't need this,
we can remove this line 
```suggestion
                            <input type="email" name="bcc" id="bcc" class="form-control" value="">
```

Reply:
Yes, sure getting the point.